Automattic/wp-calypso
# Jetpack Plans: Remove Concierge from features listing

The program is being temporarily suspended, so we need to remove any
mentions of it on the Plans page.

Test plan:

* Go to the Plans page for any Jetpack site. 
* Confirm that the Concierge feature does not appear in the Premium list.
* Switch the view to “Monthly” billing.
* Confirm that the Concierge feature does not appear in the Premium list.
